universal-application-tool-0.0.1/app/repository/ProgramRepository.java

```diff
@@ -44,4 +47,36 @@ public Program updateProgramSync(Program program) {
     ebeanServer.update(program);
     return program;
   }
+
+  public CompletionStage<Void> publishProgramAsync(Program program) {
+    return supplyAsync(
+        () -> {
+          try {
+            ebeanServer.beginTransaction();
+            program.setLifecycleStage(LifecycleStage.ACTIVE);
+            List<Program> allOldPrograms =
```

Comment:
Question (non-blocking): is there any circumstance where we expect there to be more than one program in this list?